lookout/style/typos/tests/test_corrector_utils.py

```diff
@@ -0,0 +1,141 @@
+import unittest
+import pickle
+
+import pandas
+import numpy
+from pandas.util.testing import assert_frame_equal
+
+from lookout.style.typos.utils import (flatten_data, add_context_info, rank_candidates,
+                                       filter_suggestions, suggestions_to_df,
+                                       suggestions_to_flat_df)
+
+TEST_DATA_PATH = "lookout/style/typos/tests"
+
+
+class DataTransformationsTest(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.data = pandas.read_csv(TEST_DATA_PATH + "test_data.csv", index_col=0)
```

Comment:
`"lookout/style/typos/tests" + "test_data.csv"` should be `"lookout/style/typos/teststest_data.csv"` instead of `"lookout/style/typos/tests/test_data.csv"`  - is it expected behaviour? It looks like that tests are not launched because it should raise an error.
Anyway - it's better to use `os.path.join` in such cases.